pkg/nodeconfig/nodeconfig.go

```diff
@@ -83,10 +85,6 @@ func discoverKubeAPIServerEndpoint() (string, error) {
 // NewNodeConfig creates a new instance of nodeConfig to be used by the caller.
 func NewNodeConfig(clientset *kubernetes.Clientset, ipAddress, instanceID, machineName, clusterServiceCIDR,
 	vxlanPort string, signer ssh.Signer, platform oconfig.PlatformType) (*nodeConfig, error) {
-	// Update the logger name with the VM's cloud ID. Ideally this should be the Machine name but is not available at
-	// this point.
```

Comment:
Any reason to remove this comment?

Reply:
I was thinking its not going to be accurate in the future when we are not just dealing with Machine objects.
I may have been jumping this gun with this change. Reverted it, it can be changed when those future changes are made.